lib/flash_flow/deploy.rb

```diff
@@ -143,8 +149,11 @@ def format_errors
       end
     end
 
-    def merge_or_rollback(remote, ref, pull_request_number, fix_conflicts=true)
+    def merge_or_rollback(remote, pull_request, fix_conflicts=true)
```

Comment:
The reason I pass an entire pull request in because I need it below, so this way can avoid another api calls.